doc/manifest/schema/1.1.0/defaultLocale.md

```diff
@@ -0,0 +1,287 @@
+ </details>
+
+<details>
+  <summary><b>ShortDescription</b> - The short package description</summary>
+
+  **Required Field**
+
+  This key represents the description for a package. It is intended for use in `winget show` to help a user understand what the package is.
+
+  >Note: This should be something descriptive about what the package does, and it should not simply state something like "&lt;package name&gt; installer" or "&lt;package name&gt; setup".
+</details>
+
+<details>
+  <summary><b>Description</b> - The full package description</summary>
+
+  **Optional Field**
+
+  This key represents the full or long description for a package. It is *not* currently used in the Windows Package Manager.
+
+  >Note: This was included for integration with the Microsoft Store source to provide the ability to display the full package description.
+</details>
+
+<details>
+  <summary><b>Moniker</b> - The most common package term</summary>
+
+  **Optional Field**
+
+  This key represents the most common term users would search for when installing or upgrading a package. If only one package uses this moniker, then the [install](https://docs.microsoft.com/windows/package-manager/winget/install), [list](https://docs.microsoft.com/windows/package-manager/winget/list) and [upgrade](https://docs.microsoft.com/windows/package-manager/winget/upgrade) command may match with this package. 
+  
+  >Note:Moniker is the third property evaluated when searching for a matching package.
+</details>
+
+<details>
+  <summary><b>Tags</b> - List of additional package search terms</summary>
+
+  **Optional Field**
+
+  This key represents other common term users would search for when looking for packages. Tags should be pertinent to what a user might search for when looking for a specific package.
+
+  >Note: The best practice is to present these terms in all lower case with hyphens rather than spaces.
+</details>
+
+ 
+<details>
+  <summary><b>Agreement</b> - List of package agreements</summary>
+
+   **Optional Field**
+
+   This key holds any agreements a user must accept prior to download and subsequent install or upgrade.
+
+   **Windows Package Manager Community Repository**
+
+   Agreements are only allowed in the community repository when the manifest is maintained by a verified developer.
+</details>
+  
+<details>
+  <summary><b>AgreementLabel</b> - The label for a package agreement</summary>
+    
+  **Optional Field**
+
+  This key represents the label for a package agreement.
+</details>
+
+<details>
+  <summary><b>Agreement</b> - The text for a package agreement</summary>
+    
+  **Optional Field**
+
+  This key represents the text or body of a package agreement.
+</details>
+
+<details>
+  <summary><b>AgreementUrl</b> - The URL for a package agreement</summary>
+  
+  **Optional Field**
+
+  This key represents the URL for a package agreement.
+</details>
```

Comment:
I believe that details fields can be nested within each other; Can these be nested inside the `Agreements` detail section? If they can, maybe the same can be done for `InstallerSwitches`, `Dependencies`, etc ?

Reply:
The nesting is ugly. It just appends the children to the list and isn't obvious :(

Reply:
Ahhhh, best to leave as is then

Reply:
Oh well. I got excited for a minute.

Reply:
What am I looking at? I'm not seeing any difference.